ga4gh/cli.py

```diff
@@ -1605,19 +1621,19 @@ def addReadGroupSet(self):
         Adds a new ReadGroupSet into this repo.
         """
         self._openRepo()
-        # TODO put this into a method which we can reuse.
-        try:
-            dataset = self._repo.getDatasetByName(self._args.datasetName)
-        except exceptions.DatasetNameNotFoundException:
-            raise exceptions.RepoManagerException(
-                "Cannot find dataset with name '{}'".format(
-                    self._args.datasetName))
-        dataUrl = self._args.filePath
-        indexFile = dataUrl + ".bai"
+        dataset = self._repo.getDatasetByName(self._args.datasetName)
+        dataUrl = self._args.dataFile
+        indexFile = self._args.indexFile
+        parsed = urlparse.urlparse(dataUrl)
+        if parsed.scheme == 'http':
```

Comment:
It doesn't seem to work in pysam right now. I should add FTP though, will change.